core/evancore.vue.libs/src/components/permissions/permissions.ts

```diff
@@ -83,6 +83,18 @@ class Permissions extends mixins(EvanComponent) {
     return this.sortFilter === null ? Object.keys(this.permissions) : this.sortFilter;
   }
 
+  getPropertyFieldDescription(permissions, property: string): string {
```

Comment:
I think this function could benefit from a short comment. The structure of the data is really hard to grasp from just looking at it briefly :)

Reply:
This could also be written as a computed value instead. Should make the it more performant since we are using it in multiple places.